README.md

```diff
@@ -22,7 +22,13 @@
 
 ## Installation
 
+### Choosing a location to download the repo
+
+Before running the commands, make sure you’re in the correct folder (the "current working directory") where you want to save the repo locally. You can use the `pwd` command to print the current directory, and `cd` to change the directory. You should ensure that there are no spaces at any point in the current working directory path. For example, if your top level user folder contains a space in the name, consider downloading the repo to a folder located outside of the user folder, e.g. in a new top-level `C:/apps` folder on Windows.
+
```

Comment:
nit(non-blocking): I don't think we need this duplicated here. Can assume people interacting directly with git readme likely more used to cloning repos. I'd say best to just leave the tip in the other docs